### DISCUSSION TITLE
How to use renderitem?

### DISCUSSION THREAD

**Discussion by ghost** [afconsult/quill-mention#381] [Q&A] [answered]
I'm sorry that's probably a noobish question but how can I use `renderItem`?

I'm using the `ngx-quill` library to user `quill-mention` in Angular. Everything works like a charm but I can't figure out how to render items in a certain way using `renderItem`.

```typescript 
this.modules = {
  toolbar: null,
  mention: {
    mentionDenotationChars: ['#', '@'],
    allowedChars: /^[A-Za-z\sÅÄÖÜåäöüß]*$/,
    source: (searchTerm, renderList) => {
      this.onSearch(searchTerm, renderList);
    },
    renderItem: (item, searchTerm) => {
      console.log(item);
      return `<span>test</span>`;
    }
  },
};
```

**Comment by ghost** [comment]
False alarm - was a different issue.

**Comment by yaderick** [comment, marked as answer]
const memberName = document.createElement('span');
        memberName.style.color = '#fff'; 
        memberName.textContent = item.value;
       return memberName

**Reply by badspider7** [reply]
hello,i have a lot of  components,it is a lower way to finish,is there any way to get it?
